### ISSUE TITLE
Does this project support trained by multiple GPUs?

### ISSUE THREAD

**Issue by chungjin** [Orpine/py-R-FCN#5]
I wander whether your project can be trained by multiple GPUs?
I have tried to use the command like --gpu 0,1,2  but it seems it does not support this way.
But in Dai's paper, they use 8 GPUs to train the model.

**Comment by neodooth** [comment]
I don't think this is possible, since it used python layer, and caffe does not support multi-gpu if python layer is used

**Comment by chungjin** [comment]
I think so, so they must have used the C++ and Cuda to rewrite the python layer

**Comment by bharatsingh430** [comment]
https://github.com/bharatsingh430/py-R-FCN-multiGPU

You can use this branch
